src/qt/qt_main.cpp

```diff
@@ -195,7 +195,7 @@ main(int argc, char *argv[])
     SetCurrentProcessExplicitAppUserModelID(L"86Box.86Box");
 #endif
 
-#ifndef Q_OS_APPLE
+#ifndef __APPLE__
```

Comment:
```suggestion
#ifndef Q_OS_MACOS
```

Reply:
I'll test it @lemondrops

Reply:
@lemondrops yup, your proposed change works, I'll accept it, ty!

Reply:
Thanks! I guess I should've double-checked the Qt documentation instead of trying to recall it from memory :)